<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShippingDialog</class>
 <widget class="QDialog" name="ShippingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отгрузка</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/&lt;root&gt;/icons/package.png</normaloff>:/&lt;root&gt;/icons/package.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Формирование отгрузки</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton_new_shipping_act">
           <property name="toolTip">
            <string>Создать новый акт (Ctrl+N)</string>
           </property>
           <property name="text">
            <string>Новый акт</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/&lt;root&gt;/icons/page_add.png</normaloff>:/&lt;root&gt;/icons/page_add.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+N</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_save_shipping_act">
           <property name="toolTip">
            <string>Сохранить акт (Ctrl+S)</string>
           </property>
           <property name="text">
            <string>Сохранить</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/&lt;root&gt;/icons/page_save.png</normaloff>:/&lt;root&gt;/icons/page_save.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+S</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_print_shipping_act">
           <property name="toolTip">
            <string>Распечатать акт отгрузки (Ctrl+P)</string>
           </property>
           <property name="text">
            <string>Распечатать</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/&lt;root&gt;/icons/printer.png</normaloff>:/&lt;root&gt;/icons/printer.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+P</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>№акта:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_shipping_id">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,10,1">
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Примечание</string>
           </property>
           <property name="buddy">
            <cstring>textEdit_shipping_note</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="4">
          <widget class="QTextEdit" name="textEdit_shipping_note">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="whatsThis">
            <string>Укажите здесь примечания к заказу. Например: водитель забыл шапку</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Кому отгружено</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_to</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_to"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Поиск позиций   </string>
           </property>
           <property name="buddy">
            <cstring>comboBox_shipping_candidates</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="comboBox_shipping_candidates">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>3</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Выберите номер заказа и его составляющую.</string>
             </property>
             <property name="whatsThis">
              <string>Механизм выбора компонентов заказа для добавления в акт отрузки</string>
             </property>
             <property name="maxVisibleItems">
              <number>25</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_add_to_shipping_list">
             <property name="whatsThis">
              <string>Добавить строку заказа или зака целиком</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/&lt;root&gt;/icons/package_add.png</normaloff>:/&lt;root&gt;/icons/package_add.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="popupMode">
              <enum>QToolButton::MenuButtonPopup</enum>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_remove_from_shipping_list">
             <property name="whatsThis">
              <string>Удалить строку из акта отгрузки</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/&lt;root&gt;/icons/package_delete.png</normaloff>:/&lt;root&gt;/icons/package_delete.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="5">
          <widget class="QTableWidget" name="tableWidget_shipping_list">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>5</verstretch>
            </sizepolicy>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>200</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>18</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>50</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>18</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>50</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>200</number>
           </attribute>
           <column>
            <property name="text">
             <string>№ заказа</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Позиция заказа</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Количество</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>orderedid</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Дата отгрузки</string>
           </property>
           <property name="buddy">
            <cstring>dateTimeEdit_ship</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QDateTimeEdit" name="dateTimeEdit_ship">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="whatsThis">
            <string>Укажите дату и время отгрузки</string>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy H:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Просмотр произведенных отгрузок</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Искать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_shipping_search"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_shipping_search">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/&lt;root&gt;/icons/find.png</normaloff>:/&lt;root&gt;/icons/find.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+F</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_shipping">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="action_add_one_position">
   <property name="text">
    <string>Добавить только одну позицию</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_add_all_position">
   <property name="text">
    <string>Добавить весь заказ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+*</string>
   </property>
  </action>
  <action name="actionNew_shipping_act">
   <property name="text">
    <string>new_shipping_act</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_shipping_act">
   <property name="text">
    <string>save_shipping_act</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit_shipping_search</sender>
   <signal>returnPressed()</signal>
   <receiver>toolButton_shipping_search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
